scripts/retnet.sh

```diff
@@ -2,8 +2,8 @@ python3 ../train_model.py \
     --activation-dropout 0.1 \
     --batch-size 128 \
     --checkpoints \
-    --data-dir /tmp/data \
-    --dataset-dir /tmp/data/datasets \
+    --data-dir /home/jo288/compute/retnet/data \
```

Comment:
Unless jo288 intends to run this on his account in the supercomputer, this will not work.